pkgs/build-support/compress-drv/default.nix

```diff
@@ -70,9 +70,9 @@ let
     '';
   formatsPipe = builtins.concatStringsSep "|" formats;
 in
-runCommand "${drv.name}-compressed" { } ''
+runCommand "${drv.name}-compressed" { nativeBuildInputs = [ xorg.lndir ]; } ''
```

Comment:
Curious: why do you prefer `nativeBuildInputs` above string interpolation?

Reply:
It is cross friendlier and the run commands are shorter.